src/modules/land_detector/MulticopterLandDetector.cpp

```diff
@@ -355,4 +357,22 @@ bool MulticopterLandDetector::_get_ground_effect_state()
 	       _takeoff_state == takeoff_status_s::TAKEOFF_STATE_RAMPUP;
 }
 
+bool MulticopterLandDetector::_is_close_to_ground()
+{
+	if (_vehicle_local_position.dist_bottom_valid) {
+		return _vehicle_local_position.dist_bottom < DIST_FROM_GROUND_THRESHOLD;
+
+	} else {
+		return false;
+	}
+}
+
+void MulticopterLandDetector::_set_hysteresis_factor(const int factor)
+{
+	_ground_contact_hysteresis.set_hysteresis_time_from(false, GROUND_CONTACT_TRIGGER_TIME_US * factor);
+	_landed_hysteresis.set_hysteresis_time_from(false, LAND_DETECTOR_TRIGGER_TIME_US * factor);
+	_maybe_landed_hysteresis.set_hysteresis_time_from(false, MAYBE_LAND_DETECTOR_TRIGGER_TIME_US * factor);
+	_freefall_hysteresis.set_hysteresis_time_from(false, FREEFALL_TRIGGER_TIME_US);
```

Comment:
@RomanBapst Why is a freefall less likely to happen if you're close to the ground? :thinking:

Reply:
Ha, it's not but it's still in the hysteresis factor function... Nevermind. Implementation detail.